app/views/layouts/application.html.erb

```diff
@@ -2,11 +2,22 @@
 <html>
 <head>
   <title>SnapMeUp</title>
+    <!-- Compiled and minified CSS -->
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+  <!-- Compiled and minified JavaScript -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/js/materialize.min.js"></script>
```

Comment:
JS requires should go probably at the end of the body. I ran the app got an undefined $ error as it needs to be below the javascript_include_tag on line 16 since that line is what includes JQuery